lib/string-utils.ts

```diff
@@ -71,9 +71,7 @@ export function convertToKebabCase(str: string): string {
  */
 export function getKebabCase(str?: string): string {
   return convertToKebabCase(
-    removeSpaces(
-      replaceSpecialCharacterWithSpace(replaceSpacesWithCamelCase(str ?? "")),
-    ),
+    replaceSpecialCharacterWithSpace(str),
```

Comment:
Please change this to 
`replaceSpecialCharacterWithSpace(str ?? "")`

because `str` can be undefined type

Reply:
will do , thanks